app/renderer/components/config_menu.js

```diff
@@ -39,22 +46,41 @@ module.exports = {
     </div>
     `,
 
-    methods: {
-        clearTasks() {
-            TaskConfig.clearTasks();
-            AppStatus.activeSuite = 0;
-        },
-        resetConfig() {
-            this.config.bottomBar=true;
-            this.config.animatedSpinner=true;
-        },
-        onClose() {
-            this.config.bottomBar = AppStatus.config.bottomBar;
-            this.config.animatedSpinner = AppStatus.config.animatedSpinner;
-        },
-        onSave() {
-            AppStatus.config.bottomBar = this.config.bottomBar;
-            AppStatus.config.animatedSpinner = this.config.animatedSpinner;
-        }
+  methods: {
+
+    exportTasks(){
+      let content = TaskConfig.getData() ;
+      content = JSON.stringify(content) ;
+
+        dialog.showSaveDialog({ filters: [
+
+          { name: 'gtasks', extensions: ['json'] }
+
+        ]}, function (fileName) {
+
+          if(fileName === undefined) return ;
```

Comment:
What happens if fileName is an empty string?

I'm not sure, but maybe `if(!filename)` is a better check.

Also, instead of return ;, do the folowing:
if(filename){ //or filename !== undefined
//rest of the code
}

Reply:
by default, download dialog ask you to put a file name if you let an empty string. I've seen various examples where people onli check if the filename is undefined (I'm not sure how can you set filename to undefined but...) 
If you prefer, I'll put !filename :P